### PR TITLE
Memory Managment python

### PR DESCRIPTION
**Pull request by nitishvig007**

### CODE REVIEW

File: pythonFile.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+x = 10.55
+print( 'Value of x = ', x )
+print( 'Type of x = ', type(x ))
+print( 'Memory Allocated to x = ', sys.getsizeof(x), "bytes" )
+print('Memory address at which x is stored', id(x))
+
+
+
+import matplotlib.pyplot as plt
```
Comment (sunil1510k) [inline pythonFile.py:19]: Remove this for now

File: pythonFile.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+x1 = [2, 4, 6, 8, 10]
+y1 = [10, 20, 30, 40, 99]
+
+plt.plot(x1, y1)
+plt.show()
+
+
+
+x = "10.55"
```
Comment (sunil1510k) [inline pythonFile.py:29]: Use y instead of x

File: pythonFile.py (hunk 1 above)
Comment (nitishvig007) [reply pythonFile.py:19]: This is mandatory

File: pythonFile.py (hunk 2 above)
Comment (nitishvig007) [reply pythonFile.py:29]: This is mandatory

**Pull request merged**